packages/vscode-extension/src/officeChat/commands/create/helper.ts

```diff
@@ -122,43 +120,37 @@ export async function showOfficeTemplateFileTree(
   codeSnippet?: string
 ): Promise<string> {
   const tempFolder = tmp.dirSync({ unsafeCleanup: true }).name;
-  const tempAppName = `office-addin-${crypto.randomBytes(8).toString("hex")}`;
-  const nodes = await buildTemplateFileTree(data, tempFolder, tempAppName, codeSnippet);
-  response.filetree(nodes, Uri.file(path.join(tempFolder, tempAppName)));
-  return path.join(tempFolder, tempAppName);
+  const nodes = await buildTemplateFileTree(data, tempFolder, codeSnippet);
+  response.filetree(nodes, Uri.file(tempFolder));
+  return tempFolder;
 }
 
 export async function buildTemplateFileTree(
   data: any,
   tempFolder: string,
-  tempAppName: string,
   codeSnippet?: string
 ): Promise<ChatResponseFileTree[]> {
-  const createInputs = {
+  const createInputs: Inputs = {
     ...data,
     folder: tempFolder,
-    "app-name": tempAppName,
   };
-  await vscode.commands.executeCommand(
-    CHAT_EXECUTE_COMMAND_ID,
-    CommandKey.Create,
-    TelemetryTriggerFrom.CopilotChat,
-    createInputs
-  );
-  const rootFolder = path.join(tempFolder, tempAppName);
+  const generator = new OfficeXMLAddinGenerator();
+  const context = createContextV3();
+  generator.activate(context, createInputs);
+  await generator.run(context, createInputs, tempFolder);
```

Comment:
Generator should not be called directly, should be:   fxCore.createProjectByCustomizedGenerator(createInputs, generator)

Reply:
Thanks. It has been fixed in the latest commit.